chef-server-stats/chef-server-stats

```diff
@@ -295,6 +316,7 @@ output.merge!(get_authz_stats(services))
 output.merge!(get_redis_stats(services))
 output.merge!(get_server_status)
 output.merge!(get_folsom_metrics(chef_server_12))
+output.merge!(get_pooler_metrics(chef_server_12))
```

Comment:
that looks great. But other than chef server 12 there's no other minimum version?